crmsh/ui_sbd.py

```diff
@@ -0,0 +1,551 @@
+            print(f"{node:<{max_node_len}}|{is_active_str:<{status_len}}|{is_enabled_str:<{status_len}}|{since_str_prefix}: {since}")
+        print()
+
+    def _print_watchdog_info(self):
+        padding = 2
+        max_node_len = max(len(node) for node in self.cluster_nodes) + padding
+        watchdog_sbd_re = r"\[[0-9]+\] (/dev/.*)\nIdentity: Busy: .*sbd.*\nDriver: (.*)"
+        device_list, driver_list, kernel_timeout_list = [], [], []
+
+        for node in self.cluster_nodes:
+            out = self.cluster_shell.get_stdout_or_raise_error("sbd query-watchdog", node)
+            res = re.search(watchdog_sbd_re, out)
+            if res:
+                device, driver = res.groups()
+                kernel_timeout = self.cluster_shell.get_stdout_or_raise_error("cat /proc/sys/kernel/watchdog_thresh", node)
+            else:
+                device, driver, kernel_timeout = "N/A", "N/A", "N/A"
+            device_list.append(device)
+            driver_list.append(driver)
+            kernel_timeout_list.append(kernel_timeout)
+
+        print("# Watchdog info:")
+        max_dev_len = max(len(dev) for dev in device_list+["Device"]) + padding
+        max_driver_len = max(len(driver) for driver in driver_list+["Driver"]) + padding
+        print(f"{'Node':<{max_node_len}}|{'Device':<{max_dev_len}}|{'Driver':<{max_driver_len}}|Kernel Timeout")
+        for i, node in enumerate(self.cluster_nodes):
+            print(f"{node:<{max_node_len}}|{device_list[i]:<{max_dev_len}}|{driver_list[i]:<{max_driver_len}}|{kernel_timeout_list[i]}")
+        print()
+
+    def _print_sbd_agent_status(self):
+        for node in self.cluster_nodes:
+            crm_mon_xml_parser = xmlutil.CrmMonXmlParser(node)
+            if crm_mon_xml_parser.is_resource_configured(sbd.SBDManager.SBD_RA):
+                print("# Status of fence_sbd:")
+                sbd_id_list = crm_mon_xml_parser.get_resource_id_list_via_type(sbd.SBDManager.SBD_RA)
+                for sbd_id in sbd_id_list:
+                    rc, output = self.cluster_shell.get_rc_output_without_input(node, f"crm resource status {sbd_id}")
+                    if output:
+                        print(output)
+                return
+
+    def _print_sbd_cgroup_status(self):
+        scripts_in_shell = '''#!/bin/bash
+cgroup_procs_file="/sys/fs/cgroup/system.slice/sbd.service/cgroup.procs"
+if [ ! -f "$cgroup_procs_file" ]; then
+    exit
+fi
+pids=$(cat "$cgroup_procs_file")
+for pid in $pids; do
+    cmdline_file="/proc/$pid/cmdline"
+    if [ -f "$cmdline_file" ]; then
+        cmdline=$(tr '\0' ' ' < "$cmdline_file")
+        if [[ "$cmdline" == *"slot:"* ]]; then
+            echo "├─$pid \"$cmdline\""
+        fi
+    fi
+done
+        '''
+        for node in self.cluster_nodes:
+            out = self.cluster_shell.get_stdout_or_raise_error(scripts_in_shell, node)
+            if out:
+                print(f"# Status of sbd process on {node}:")
```

Comment:
```suggestion
                print(f"# Status of the sbd disk watcher process on {node}:")
```
And, this information should no be printed out for the diskless-sbd.